{"name": "test_error_app_key", "status": "passed", "description": "错误的app_key参数", "start": 1599985560141, "stop": 1599985560162, "uuid": "e6c39c81-27d1-4b0a-820c-a1f22adb300b", "historyId": "c65a9a8f558881df192a38af12688d4e", "testCaseId": "b02efde331dcd4e27dafc6ea28e8f72c", "fullName": "case.test_login_pytest.Test_Login_user#test_error_app_key", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "Test_Login_user"}, {"name": "host", "value": "V_LFJJLIU-NB0"}, {"name": "thread", "value": "14372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login_pytest"}]}